server/readme.md

```diff
@@ -45,9 +45,41 @@ The following environment variables are required:
       "syslog_drain_url": "",
       "tags": [],
       "volume_mounts": []
+    },
+    {
+      "credentials": {
+        "cert": "40a4ce56c7494ed37649/test.pdf",
+        "entrypoint": "https://idp.int.login.gov/api/saml/auth",
+        "issuer": "urn:gov:gsa:SAML:2.0.profiles:sp:sso:usda-forestservice:epermit-dev",
```

Comment:
This is going to change from env to env so put something more general here like login-gov issuer from dasbhoard.